assets/src/components/detours/detourFinishedPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import { Button, Form, OverlayTrigger, Tooltip } from "react-bootstrap"
+import * as BsIcons from "../../helpers/bsIcons"
+import { Panel } from "./diversionPage"
+
+export const DetourFinishedPanel = ({
```

Comment:
**Non-blocking:** I think it would be good to add a storybook story for this.

Reply:
Sounds good to me!
I need #2432 for it to look correct. So I'm debating whether to merge this and add the story as a followup or to block this PR on it

Reply:
Oops, one moment

Reply:
PTALA 451d3e0caf2e816fd64b1ea0dc23f57211297c71 🙏🏻

Reply:
There's also a9bff7e6915ec6426a2e98edaf9b02b1e9f81149 and 0fcce1b216eec2bc586be0238c28425bfd581ad5 because I had to patch an issue with the CSS

Reply:
I'm gonna go ahead and merge and we can revisit anything else later, here's the [storybook story link](https://65047eaabeb50e4310dc692b-qyylnydajb.chromatic.com/?path=/story/skate-detours-detourfinishedpanel--story)!